doc/release-notes-bucash.md

```diff
@@ -32,162 +33,364 @@ If you are upgrading from a release older than 1.1.2.0, your UTXO database will
 to a new format. This step could take a variable amount of time that will depend
 on the performance of the hardware you are using.
 
-Other than that upgrading from a version lower than 1.5.0.1 your client is probably stuck
+Other than that upgrading from a version lower than 1.5.0.0 your client is probably stuck
 on a minority chain and need some manual intervention to make so that it follow the majority
 chain after the upgrade. For more detail please look at `reconsidermostworkchain` RPC commands.
 
-Main Changes in 1.5.1
+Main Changes in 1.6.0
 ---------------------
 
-- 10x transaction processing performance
-- Extended version message: [xversion](https://github.com/BitcoinUnlimited/BitcoinUnlimited/blob/release/doc/xversionmessage.md)
-- Adding a checkpoint at height 556767 both for the SV and the BCH chain
-
-Bug Fixed since 1.5.1.0-rc2
-------------------------
-
-- Documentation fixes
-- Make it possible to disable graphene and xthin preferential timer
-- Switch ctor on earlier in the init process if we are on the BCH chain
-- Improve Graphene decode rates by padding IBLT
-- Improve xversion QA testing
-- General improvements and fixes to the QA test framework
+- Add SCRIPT_ALLOW_SEGWIT_RECOVERY (May 15th upgrade)
+- Schnorr signature scheme (May 15th upgrade)
+- RPC enhancements
```

Comment:
@Greg-Griffith can you summarize the changes in the batch-tx-rpc branch here

Reply:
That PR added 3 new rpc commands:
- getrawblocktransactions
  - getrawblocktransactions takes the block hash and a protocol id (optional) and returns the raw transaction data of all transactions in the block. If a protocol id is supplied then it will not return any transactions that do not have an OP_RETURN that meets the protocol id criteria. Use * as a wildcard for any id.

- getrawtransactionssince
  - getrawtransactionssince takes the blockhash, block count, and a protocol id (optional) and returns the raw transaction data for <count> blocks starting with <blockhash> and moving towards the tip.  If a protocol id is supplied then it will not return any transactions that do not have an OP_RETURN that meets the protocol id criteria. Use * as a wildcard for any id.

- gettxoutproofs
  - gettxoutproofs takes a list of transaction hashes and a blockhash. it returns a hex-encoded proof for each transaction in the given list of transaction hashes that is inside the block with hash <blockhash>..

Reply:
Will add these new 3 rpc calls to the list of changes. I don't think we should add also the entire description for all of them, thou.